app/models/user.rb

```diff
@@ -7,5 +7,6 @@ class User < ApplicationRecord
   validates :password, presence: { require: true }
   validates :password_confirmation, presence: { require: true }
 
+  has_many :friendships
```

Comment:
I'm not sure if it needs this, but we can look at Rubocop and see if it's happy with it